pkg/apis/storagepool/config/cns.vmware.com_storagepools.yaml

```diff
@@ -1,10 +1,10 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Why do we need this change ?

Reply:
this got updated during `make build-bins` after upgrading go lang version.

Reply:
revered changes made in this file. controller-gen binary on the machine where I was building binary was older.